rllib/core/rl_module/marl_module.py

```diff
@@ -301,7 +301,7 @@ def _forward_train(
         Returns:
             The output of the forward_train pass the specified modules.
         """
-        return self.__run_forward_pass("forward_train", batch, **kwargs)
```

Comment:
I had to make this change since tensorflow function tracing does not support tracing through functions that have mangled names.